<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExceptionFormClass</class>
 <widget class="QWidget" name="ExceptionFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExceptionForm</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="exceptionNameLabel">
      <property name="text">
       <string>Exception name (*)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="exceptionNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	border: 1px solid #ccc;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>129</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="paidHourLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Paid hours</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="paidHourTime">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximumTime">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>67</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="paidCoeffLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Paid coefficient [0, 3]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="paidCoeffSpinBox">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="alwayPayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #EAEFF5;
	border: none;
}</string>
   </property>
   <property name="text">
    <string>Always Pay</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	border: 1px solid #ccc;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_11">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>54</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="workCoeffLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Work coefficient [0, 10]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="workCoeffSpinBox">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="isOverTimeCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Include working hours in the overtim calculations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/check.png</normaloff>../icons/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/close.png</normaloff>../icons/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="extralabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #BFD0EA;
	border: none;
}</string>
   </property>
   <property name="text">
    <string>Extra Working Hour Paid</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>alwayPayLabel</zorder>
  <zorder>isOverTimeCheckBox</zorder>
  <zorder>btnOk</zorder>
  <zorder>btnCancel</zorder>
  <zorder>extralabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
